### ISSUE TITLE
Unclear where to report bugs on Arduino Create

### ISSUE THREAD

**Issue by matthijskooijman** [arduino/arduino-create#1]
I tried figuring out where to report bugs. Through the Arduino repo list I found this repo, but I doubt it is the right one (given there's nothing here). I looked around the Arduino Create interface itself, and at https://www.arduino.cc/en/Main/ContactUs but neither showed anything.

**Comment by 00alis** [comment]
Hi @matthijskooijman,
we are currently using the forum for bug reporting (even thou it has not been designed for it of course). https://forum.arduino.cc/index.php?board=101.0

@mastrolinux was this repo created recently to get the community the opportunity to use Github or is it something old?

**Comment by matthijskooijman** [comment]
Yeah, I just noticed the forum in the welcome mail. Might be good to mention it elsewhere, the ContactUs page is probably a good one, since it already lists a lot of other places too?
